packages/example-application/src/formik.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useState, useCallback } from 'react';
+import * as GovUK from 'govuk-react';
+import { Link } from 'react-router-dom';
+
+import { Formik, Form, Field as FormikField } from 'formik';
+
+const Field = ({ component: Component, ...props }) => (
+  <FormikField {...props}>{({ field, meta }) => <Component {...props} input={field} meta={meta} />}</FormikField>
+);
+
+const Checkbox = ({ input, ...props }) => <GovUK.Checkbox {...input} {...props} />; //eslint-disable-line
+const DateField = ({ meta, input: { onChange, onBlur, ...input }, ...props }) => (
+  <GovUK.DateField
```

Comment:
this is a bit of a mess, the DateField in particular, though the point of this is to illustrate issues with the component API that need to be addressed in 0.10

Reply:
Maybe create an issue to improve?